packages/node/src/node.ts

```diff
@@ -350,7 +349,11 @@ export class Hatchify {
   getHatchifyModelNameForEndpointName(endpointName: string): false | string {
     // Validate if endpoint name is lowercase
     if (endpointName === endpointName.toLowerCase()) {
-      const singular = this._pluralToSingularModelNames.get(endpointName)
+      //Endpoints follow kebab-case convention; must convert to flat case to compare
+      const flatCaseEndpointName = endpointName.replace("-", "")
```

Comment:
This would be the opposite, actually. We want to convert from kebab to flat case (as in: "kebab-case" > "kebabcase"). I believe lodash doesn't have such a function, but I may be wrong.